openfga/v1/openfga_service.proto

```diff
@@ -1687,6 +1687,15 @@ message ListStoresRequest {
     (validate.rules).string.max_bytes = 5120,
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "\"eyJwayI6IkxBVEVTVF9OU0NPTkZJR19hdXRoMHN0b3JlIiwic2siOiIxem1qbXF3MWZLZExTcUoyN01MdTdqTjh0cWgifQ==\""}
   ];
+
+  string name = 3 [
+    (google.api.field_behavior) = OPTIONAL,
```

Comment:
I think `OPTIONAL` is the default and doesn't need to be stated.

That said, it doesn't hurt to have it specified.

Reply:
Yeah the default is optional but I think it better to rely on explicit behavior instead of implicit